<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAttendannce</class>
 <widget class="QMainWindow" name="FaceAttendannce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceAttendannce</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>480</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="widgetLb" native="true">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(49, 50, 54,63);
border-radius :10px;</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/yes.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
background-color: rgba(49, 50, 54,63);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>认证成功</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="headpicLb">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>266</width>
       <height>266</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>266</width>
       <height>266</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>266</width>
       <height>266</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/crc.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="videoLb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>videoLb</zorder>
    <zorder>widgetLb</zorder>
    <zorder>headpicLb</zorder>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>320</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>480</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	border: 1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}

QWidget#titleLb{
	
	font: 14pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QWidget#headLb{
	border-radius:75px;
}</string>
    </property>
    <widget class="QLabel" name="titleLb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>人脸识别考勤系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headLb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>161</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border: 1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgba(255, 255, 255,0.8);</string>
          </property>
          <property name="text">
           <string>工号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numberEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>111</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border: 1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgba(255, 255, 255,0.8);</string>
          </property>
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>111</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border: 1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,10">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgba(255, 255, 255,0.8);</string>
          </property>
          <property name="text">
           <string>部门：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="departmentEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>111</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>280</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>230</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border: 1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49,0.9);
}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,10">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgba(255, 255, 255,0.8);</string>
          </property>
          <property name="text">
           <string>时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>8</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>111</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
